projects/models.py

```diff
@@ -9,12 +9,18 @@ class Project(models.Model):
     title = models.CharField(max_length=200)
     description = models.TextField()
     max_members = models.IntegerField()
-    difficulty = ChoiceField(choices=['beginner', 'intermediate', 'advanced'])
+    difficulty_choices = (
+        ('beginner', 'beginner'),
+        ('intermediate', 'intermediate'),
+        ('advanced', 'advanced')
+        )
```

Comment:
jose pointed me, enumfield that looks interesting for such cases https://pypi.python.org/pypi/django-enumfield/, just have a look do not get crazy to use it